chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -218,6 +218,7 @@
 
         when os =~ /^(open)?suse/
           append << "install=#{install_url} autoyast=#{node_url}/autoyast.xml"
+          append << "ifcfg=dhcp4 netwait=60"
```

Comment:
exactly. the ifcfg option is new in SLE12, so is ignored in SLE11 and the netwait does not hurt too